pkg/multisort/compare.go

```diff
@@ -20,7 +20,8 @@ const (
 )
 
 // Compare compares values according to multisort criteria.
-func Compare[O constraints.Ordered](a O, b O, descending bool) CompareResult {
+// TODO: can be replaced by cmp.Compare
```

Comment:
How can this be replaced? I don't think it's easily possible in our multisort algorithm.

Reply:
A descending compare would be `-1 * cmp.Compare(l, r)`

Reply:
It's not a typical compare return value though, it returns less or "not equal".

Reply:
OK, i will remove the comment

Reply:
I mean, you can try if you want. When it passes the tests, it should be alright.

Reply:
And it returns `0` if the values are neither less or not not-equal. If we change the numbers behind the constants (`Less = -1` and `NotEqual = 1`) it would behave like `cmp.Compare`.

Reply:
But the cmp.Compare does not have the `descending bool` argument